builds/jazz-build-module/sls-app/sbr.groovy

```diff
@@ -1120,18 +1123,38 @@ def prepareServerlessYml(aConfig, env, configLoader, envDeploymenDescriptor) {
     deploymentDescriptor = aConfig['deployment_descriptor']
   }
   try {
-    def appContent = readFile('application.yml').trim()
-    if(!appContent.isEmpty()) deploymentDescriptor = appContent
-    } catch(e) { // TODO to catch the type error
-      echo "The application.yml does not exist in the code. So the default value from config will be used.$e"
+    def appContent = readFile('application.yml').trim() // copy of the user serverless.yml
+    if(!appContent.isEmpty()) {
+      echo "User supplied serverless.yml is being used."
+      deploymentDescriptor = appContent
     }
+  } catch(e) { // TODO to catch the type error
+      echo "The user supplied serverless.yml does not exist in the code. So the default value from config will be used: ${e}"
+  }
 
     def doc = deploymentDescriptor  ? readYaml(text: deploymentDescriptor ) : [:] // If no descriptor present then simply making an empty one. The readYaml default behavior is to return empty string back that is harful as Map not String is expected below
 
+    def envKinesisStreamArn = null
+    
+    switch (env)
```

Comment:
This is incorrect. We always use `configLoader.JAZZ.PLATFORM.AWS.KINESIS_LOGS_STREAM.PROD` for user services.

`configLoader.JAZZ.PLATFORM.AWS.KINESIS_LOGS_STREAM.STG` & `configLoader.JAZZ.PLATFORM.AWS.KINESIS_LOGS_STREAM.DEV` is for developing services that read messages off of the stream like dev versions of log-streamer services.

Reply:
done